script.js

```diff
@@ -218,4 +218,18 @@ projectButtons.forEach((button) => {
     modalMobile.classList.remove('display-none');
     modalDesktop.classList.remove('display-none');
   });
+});
+
+const form = document.getElementById('contact-form');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const { email } = form.elements;
+  const emailRegex = /[A-Z]/;
+  const message = form.querySelector('small');
+  if (emailRegex.test(email.value)) {
+    message.textContent = 'You can only use Lowercase, please modify the email address';
```

Comment:
- [ ] Great job. everything works well. please kindly ensure that the error message come up beside the submit button so it can easily be noticed. great work
![Screenshot (530)_LI](https://user-images.githubusercontent.com/65298048/195922756-2cd0aaad-bf2e-4a5f-a6b6-9fb29773c5ea.jpg)